css/style.css

```
@@ -0,0 +1,31 @@
+body {
+  margin: 0;
+  padding: 0;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 20px;
```

```suggestion
  font-size: 1.25rem;
```
Please consider using fluid measurements such as rem. They are much better for responsive design because they are relative measurements. 
px, pixels are absolute measurements. rem is the font size of the root element default size of 16px. When you resize the screen or viewport, the rem adjusts.

---

I love how you explained the reason for the change :+1: . Students reading your inline comments will be more motivated to implement the changes in the future :rocket: